docs/content/latest/api/ycql/expr_fcall.md

```diff
@@ -87,7 +88,34 @@ CAST function converts the value returned from a table column to the specified d
 | `TIMESTAMP` | `DATE`, `TEXT` |
 | `TIMEUUID` | `DATE`, `TIMESTAMP` |
 
-### Examples
+## WriteTime function
+We can get the timestamp when the column/row was writen.
```

Comment:
What units is the writetime function return the value in?

writen -> written 

Wording wise, there needs to be a bit more formal wording style. Look at other documentation for functions on our or other product sites.

For example, something like:

The `writetime` function returns the timestamp in <whatever-units> when a column was written.